### ISSUE TITLE
Have two different packages for Hspec and Tasty

### ISSUE THREAD

**Issue by michivi** [alexbiehl/testcontainers-hs#19]
### Question

Would it be possible to have two different packages, one for Hspec and one for Tasty?

Currently, when one chooses to use Hspec (resp. Tasty), they also have to compile Tasty (resp. Hspec) as `testcontainers-hs` uses both as dependencies. But they won't actually use it.

Perhaps having something like this could help:

- The `testcontainers-hs-core` package would have all the definitions required to use `testcontainers-hs`, whether we're a Hspec user or Tasty user.
- The `testcontainers-hs-hspec` package for Hspec users
- The `testcontainers-hs-tasty` package for Tasty users

Do you think something like that would be doable?
Thanks in any case 🙂

**Comment by michivi** [comment]
Right, but it still feels unnecessary to build tasty and its dependencies when we're using hspec. In CI, we would also use unnecessary storage for the compiled binary, though surely not significantly to be important. In any case, I guess this sure sounds low-priority as nothing is broken. If I have time, I'll try to tackle it 🙂

**Comment by alexbiehl** [comment]
Sounds good!
